docs/source/en/_toctree.yml

```diff
@@ -29,7 +29,7 @@
     - local: using-diffusers/schedulers
       title: Load and compare different schedulers
     - local: using-diffusers/custom_pipeline_overview
-      title: Load community pipelines
+      title: Load community pipelines and components
```

Comment:
We're not adding a separate doc page here. 

See: https://github.com/huggingface/diffusers/pull/5556/files#r1376500960/